tests/includes/Sync/Test_Sync_Journal_Observation.php

```
@@ -278,15 +278,24 @@ public function test_hpos_order_trash_and_untrash_append_delete_then_present_row
 
 			wp_cache_flush();
 			$cursor = $this->journal->head_sequence();
+			$untrash_rows_before_cleanup = null;
+			$observe_cleanup = function ( $_meta_id, $object_id, $meta_key ) use ( $order_id, $cursor, &$untrash_rows_before_cleanup ): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$_meta_id'. (undefined)

(UnusedFormalParameter)